test/src/tiny_uart_double.cpp

```diff
@@ -16,6 +16,9 @@ static void send(i_tiny_uart_t* _self, uint8_t byte)
   if(self->automatic_send_complete) {
     tiny_uart_double_trigger_send_complete(self);
   }
+  if(self->echoing) {
```

Comment:
```suggestion

  if(self->echoing) {
```

This helps my feeble brain from reading this as an else-if